### ISSUE TITLE
Iframes?

### ISSUE THREAD

**Issue by mijnnaamisramon** [Artem-Schander/quill-paste-smart#8]
Hey! The package is working nicely! But when I paste an iframe (like for a YouTube embed).... it pastes the content as text (as you would guess)... but now I have a bunch of text that looks like an iframe, but isn't. I've been trying to use clipboard addmatchers to detect the pasting of an iframe and only paste the src of the iframe, without the rest... but so far, no luck. Does this package have some options to make this work better? :)

**Comment by artem-schander** [comment]
Hi @mijnnaamisramon ,

TL;DR
No. this package is not meant to make HTML out of a regular (or escaped HTML) string.

Long answer:
I think you get it a little wrong. As far as I understand it correctly, the iframe string you have copied to your clipboard is not really HTML, it may look like valid HTML but it is kind of escaped. So all the HTML entities are encoded.
F.A. <iframe> is in fact `&lt;iframe&gt;`

Besides that, this package would let you paste a fully functional iframe, you just need to add the video module to the toolbar. See: https://nnzxi.csb.app/
Or alternatively add `iframe` to the `clipboard.allowed.tags` list. See [custom configuration](https://github.com/Artem-Schander/quill-paste-smart#configuration)

Also it could be worth a read [how to handle HTML paste](https://stackoverflow.com/a/48155904/2891211)

I hope this answer helps somehow. I'm closing this issue for now. Feel free to reopen it if you think we are not done.
